app/src/main/java/com/onboarding/parkingsystemjava/mvp/contract/MainActivityContract.java

```diff
@@ -4,14 +4,19 @@
 
     interface MainActivityPresenter {
         void onSetParkingButtonPressed();
+
+        void setParkingLots(int lots);
     }
 
     interface MainActivityModel {
-        void setParkingLots();
+        void setParkingLots(int setLots);
+
         int getParkingLots();
     }
 
     interface MainActivityView {
         void showParkingLotsAvailable(int parkingLots);
+
+        void showSetParkingLotsDialogFragment();
```

Comment:
this show or this set?

Reply:
line 18 method was for the previous pr, forgot to delete it.

Reply:
still misnamed